test/parallel/test-process-warnings.mjs

```diff
@@ -0,0 +1,148 @@
+import { spawnPromisified } from '../common/index.mjs';
+import * as fixtures from '../common/fixtures.mjs';
+import { describe, it } from 'node:test';
+import assert from 'node:assert';
+
+const fixturePath = fixtures.path('disable-warnings.js');
+const fixturePathWorker = fixtures.path('disable-warnings-worker.js');
+const dep0025Message = /\(node:\d+\) \[DEP0025\] DeprecationWarning: sys is deprecated\. Use util instead\./;
+const dep0094Message = /\(node:\d+\) \[DEP0094\] DeprecationWarning: assert\.fail\(\) with more than one argument is deprecated\. Please use assert\.strictEqual\(\) instead or only pass a message\./;
+const experimentalWarningMessage = /\(node:\d+\) ExperimentalWarning: vm\.measureMemory is an experimental feature and might change at any time/;
```

Comment:
Ideally, changing the text of these messages wouldn’t break this unrelated test.

```suggestion
const dep0025Message = /\(node:\d+\) \[DEP0025\] DeprecationWarning/;
const dep0094Message = /\(node:\d+\) \[DEP0094\] DeprecationWarning/;
const experimentalWarningMessage = /\(node:\d+\) ExperimentalWarning/;
```

What about when `vm.measureMemory` leaves experimental? Though I guess there’s probably no experimental feature that’s assured to stay experimental forever, other than `async_hooks` which I think doesn’t emit a warning.

Reply:
Good point - I could just emit an Experimental warning manually?

Reply:
I thought about this some more, and am switching to all manual `process.emitWarning` calls. There's always a chance things change in the future for the deprecation and experimental warnings. This makes the test much more resilient.